packages/z2s/src/compiler.pg-test.ts

```diff
@@ -587,7 +593,7 @@ describe('compiling ZQL to SQL', () => {
         .related('comments', q =>
           q.where('text', 'ILIKE', '%2%').related('author'),
         );
-      const sqlQuery = formatPg(
+      const sqlQuery = formatPgInternalConvert(
```

Comment:
Would be good to update the comment where filter to be on `createdAt` instead of text so that we exercise a timestamp filter.